app/assets/src/components/SampleUpload.jsx

```diff
@@ -33,6 +34,7 @@ class SampleUpload extends React.Component {
     const adminGenomes = this.hostGenomes.filter((g) => {
       return g.name.toLowerCase().indexOf('test') >= 0;
     });
+    console.log('Fuck', JSON.stringify(this.props.projects));
```

Comment:
just remove this line and it'll lgtm :sweat_smile:

Reply:
🤦‍♂️